draft-scenarios/clash/gold_rush.tex

```diff
@@ -0,0 +1,164 @@
+  \item \textit{Obelisk Pile}: remaining non-Relic Artifacts that grant resources (including those with one of the options) and have no ongoing effect
+  \item \textit{Remaining Artifacts}: remaining Artifacts that grant resources 
+\end{enumerate}
+  
+These removed Cards will be available at specific locations throughout the Scenario.
+
+\vfill
+
+\subsection*{\MakeUppercase{Additional Rules}}  
+
+\begin{table*}[b!]  
+  \hommtable[]{22}{  
+    \centering  
+    \begin{tabularx}{\linewidth}{p{0.25\linewidth}p{0.71\linewidth}}  
+      \darkcell{Tile and Location} & \darkcell{Additional Rules} \\  
+      \darkcell[1.3]{Obelisk} 
+      & \lightcell[1.3]{Pick one Artifact from the \textit{Obelisk Pile}, or roll 2 \svg{treasure-note} and choose one result.} \\  
+      \darkcell[1]{N1 Witch Hut}  
+      & \lightcell[1]{Instead of the standard effect, gain one of the \textit{Estates} Abilities.} \\  
+      \darkcell[1]{\#C1 Warrior Tomb}  
+      & \lightcell[1]{Instead of searching for an Artifact, pick one \textit{Statue of Legion} piece.} \\  
+      \darkcell[1.3]{\#C1/C2 Shrine of Magic Incantation}  
+      & \lightcell[1.3]{Instead of searching for a Spell, gain one \textit{View Air} Spell.} \\  
+      \darkcell[1.6]{\#C1 Settlement}  
+      & \lightcell[1.6]{Combat with Neutral Units cannot be skipped*. The first player to visit gains all remaining Artifacts from the \textit{Obelisk} and \textit{Remaining Artifacts} piles.} \\  
+      \darkcell[1.3]{C2 Grail}  
+      & \lightcell[1.3]{Combat with Neutral Units cannot be skipped*. Take the Grail Token and lose all remaining \svgeven{movement-note}.} \\  
+    \end{tabularx}  
+    \medskip  
+  }  
+  {\footnotesize * Including Quick Combat. Bypassing without visiting the Field (such as with the \textit{Pathfinding} Ability) is still allowed.}  
+\end{table*}  
+
+\begin{itemize}
+  \item At the start of certain Rounds, players must send a \textbf{tranche} to their lieges. The tranche value is expressed in \svg{gold}, but may be partially or entirely paid in other resources using these exchange rates: 1 \svg{building_materials} equals 2 \svg{gold} and 1 \svg{valuables} equals 4 \svg{gold}, with no change given. In Alliance mode, allied players collectively pay a single tranche. Any player or team unable to pay loses the game immediately.  
+  \item The difficulty of all Fields on the \#C1 Tile is lowered by one Level (effectively making it a ``V-VI'' Tile).  
+  \item Certain locations have special rules, typically allowing players to acquire resource-generating Cards - refer to the table below.
+  \item A Main Hero carrying the Grail Token is not protected by the \textit{Sanctuary} effect.
+  \item The Surrendering cost for a Main Hero carrying the Grail Token is increased to 30 \svg{gold}.
+  \item When defeating an enemy Main Hero, the victorious player may, instead of taking 5 \svg{gold}, choose to take either the Grail Token or one Artifact from the opponent's Discard Pile.
```

Comment:
```suggestion
  \item When defeating an enemy Main Hero, the victorious player may, instead of taking \mbox{5 \svg{gold}}, choose to take either the Grail Token or one Artifact from the opponent's Discard Pile.
```
This prevents the line split between the 5 and the glyph.